migrations/2025-04-10-095823_add_platform_context_in_organization/up.sql

```diff
@@ -0,0 +1,5 @@
+-- Your SQL goes here
+ALTER TABLE organization ADD COLUMN IF NOT EXISTS organization_type VARCHAR(64) NOT NULL DEFAULT 'standard';
+ALTER TABLE organization ADD COLUMN IF NOT EXISTS platform_merchant_id VARCHAR(64);
+
+ALTER TABLE merchant_account ADD COLUMN IF NOT EXISTS merchant_account_type VARCHAR(64) NOT NULL DEFAULT 'standard';
```

Comment:
we should not add NOT NULL based alter query, and make default in database instead use default value in code and have nullable value in database

Reply:
we can handle the null cases in code itself